android/build.gradle

```diff
@@ -53,7 +53,7 @@ android {
 }
 
 dependencies {
-  compile 'com.facebook.react:react-native:+'
-  compile "com.google.android.gms:play-services-base:9.4.0"
-  compile 'com.google.android.gms:play-services-maps:9.4.0'
+  compile "com.facebook.react:react-native:+"
+  compile "com.google.android.gms:play-services-base:+"
+  compile "com.google.android.gms:play-services-maps:+"
```

Comment:
Just curious, what's the advantage of explicitly setting it versus using + (similar to what's done for react-native)?

Looking at the most popular react-native libraries that use Google Play Services the rest of them use +, which makes it difficult to use them with react-native-maps.